src/components/Row/coinRow.js

```diff
@@ -138,5 +141,49 @@ export function CoinRow({ activeCoin, coinList, updateSelectedCoins }) {
 		);
 	});
 
-	return <ScrollRow>{coinButtons}</ScrollRow>;
+	const [maxScrollLeft, setMaxScrollLeft] = useState(0);
+
+	useEffect(() => {
```

Comment:
I don't think we need this use effect here (just doing initialization), move the inside of the set function to the useState(...) call. useState takes the initial state as an argument.

Reply:
I tried moving the set function inside of the useState() call, but it errors out every time because it cannot access scroll.current before initialization. I think the useEffect() needs to stay for that reason

Reply:
Okay, yea that makes sense 👍